api/jobs.go

```diff
@@ -151,6 +156,17 @@ func (api *API) GetJobsHandler(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (api *API) setUpPagination(w http.ResponseWriter, offsetParam string, limitParam string, ctx context.Context) (int, int, bool) {
```

Comment:
Can I suggest returning the error and deal with it in `GetJobsHandler`?

```suggestion
func (api *API) setUpPagination(w http.ResponseWriter, offsetParam string, limitParam string, ctx context.Context) (int, int, error) {
```

Reply:
yes that makes sense - done